deploy/README.md

```diff
@@ -42,6 +42,9 @@ kubectl exec -it ${NMS_POD} -- yarn setAdminPassword magma-test admin admin
 
 Setup Ansible:
 ```bash
-sudo apt install -y ansible
+sudo apt update
+sudo apt install software-properties-common
+sudo add-apt-repository --yes --update ppa:ansible/ansible
+sudo apt install ansible
 ansible-galaxy collection install shubhamtatvamasi.magma --force-with-deps
```

Comment:
I think that this command won't work after we have installed latest ansible. Can you confirm?

Reply:
yes

Reply:
this is only for installation if there is no ansible installed previously. Suppose, if ansible is installed but we want to upgrade it to latest version then we have to add one more command above this to remove previously installed ansible and above command then install latest version

Reply:
I am talking about `collection install` command.

Reply:
ok. Yes it won't work

Reply:
Then update it too.

Reply:
ok

Reply:
hi,  _collection install_ work properly.